api/mesh/v1alpha1/dataplane.proto

```diff
@@ -48,6 +48,9 @@ message Dataplane {
       // Port of the service that requests will be forwarded to.
       uint32 servicePort = 4;
 
+      // Address of the service that requests will be forwarded to.
```

Comment:
also `127.0.0.1 by default. Since Kuma DP should be deployed next to service`

or something like this